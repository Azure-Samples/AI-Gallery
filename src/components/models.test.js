import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer'
import Models from './models';


test('renders a snapshot of Models', () => {
    const tree = renderer.create(<Models/>).toJSON();
    expect(tree).toMatchSnapshot();
})