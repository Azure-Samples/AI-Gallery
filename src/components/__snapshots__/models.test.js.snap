// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders a snapshot of Models 1`] = `
<div>
  <div
    class="menu-wrapper"
    id="model-wrapper"
  >
    <div>
      <ul
        className="menu models"
        id="models-menu"
      >
        <li
          className="modelsitem item"
        >
          <div
            className={
              Array [
                "github-card",
              ]
            }
            data-github="Azure-Samples/AI-Gallery-InceptionV3"
            data-theme="medium"
          />
        </li>
        <li
          className="modelsitem item"
        >
          <div
            className={
              Array [
                "github-card",
              ]
            }
            data-github="Azure-Samples/AI-Gallery-TinyYolo"
            data-theme="medium"
          />
        </li>
        <li
          className="modelsitem item"
        >
          <div
            className={
              Array [
                "github-card",
              ]
            }
            data-github="onnx/models"
            data-theme="medium"
          />
        </li>
      </ul>
      <div
        class="paddles"
      >
        <button
          class="left-paddle paddle hidden"
          id="models-left"
          onClick={[Function]}
        >
          &lt;
        </button>
        <button
          class="right-paddle paddle"
          id="models-right"
          onClick={[Function]}
        >
          &gt;
        </button>
      </div>
    </div>
  </div>
</div>
`;
